content/src/content/jcr_root/apps/core/wcm/components/navigation/v1/navigation/_cq_dialog/.content.xml

```diff
@@ -48,6 +48,15 @@
                                         jcr:primaryType="nt:unstructured"
                                         sling:resourceType="granite/ui/components/coral/foundation/container">
                                         <items jcr:primaryType="nt:unstructured">
+                                            <label
+                                                jcr:primaryType="nt:unstructured"
+                                                sling:resourceType="granite/ui/components/coral/foundation/form/textfield"
+                                                fieldDescription="Value of aria-label attribute for the navigation. Should be specified if more than one naviagtion is used on a page."
```

Comment:
Would improve this a little:
"Value of an aria-label attribute for the navigation. Should be added there is more than one navigation on the page."

Reply:
Updatede the description